custom-metrics-stackdriver-adapter/README.md

```diff
@@ -1,8 +1,8 @@
 # Custom Metrics - Stackdriver Adapter
 
 Custom Metrics - Stackdriver Adapter is an implementation of [Custom Metrics
-API] based on Stackdriver per-pod metrics. It's purpose is to enable pod
-autoscaling based on Stackdriver custom metrics.
+API] and [External Metrics API] based on Stackdriver per-pod metrics. It's
+purpose autoscaling based on Stackdriver custom metrics.
```

Comment:
That paragraph sounds like EM API is based on per-pod metrics. Also last sentence seems to miss a few words (and also in last sentence it should be `Its` not `It's`).

Reply:
Fixed.